templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
       });
 
       chai.should();
-      const expect = chai.expect;
+      window.expect = chai.expect;
     </script>
 
     <!-- Package-specific tests matching packages/<pkg>/test/browser/*.js -->
 
     <script>
     describe('{{pkg}} > test/sw/*.js', function() {
-      {{#each scripts}}
-      it(`should run '{{this}}' sw tests`, function() {
```

Comment:
These template changes work both for packages whose tests I changed as well as packages whose tests I didn't change, so I think it's good enough for now.